test/general/tizen_plugins_test.dart

```diff
@@ -52,14 +55,37 @@ flutter:
         fileName: some_dart_plugin.dart
 ''');
     pubspecFile.writeAsStringSync('''
+name: plugin_test
 dependencies:
   some_dart_plugin:
     path: ${pluginDir.path}
+''');
+    packageGraphFile.writeAsStringSync('''
+{
+  "roots": ["plugin_test"],
+  "packages": [
+    {
+      "name": "plugin_test",
+      "dependencies": ["some_dart_plugin"]
+    },
+    {
+      "name": "some_dart_plugin",
+      "dependencies": []
+    }
+  ],
+  "configVersion": 1
+}
```

Comment:
flutter/packages/flutter_tools/test/general.shard/package_graph_test.dart
Upstream, it seems like JSON content is being written using jsonEncode. Do we need to update our test code as well?

Since the same file creation is repeated, can it be made into a function?
package_graph_test.dart
```
...
void writePackageGraph(List<Package> graph) {
...
```

Reply:
Thank you for the review.
Actually, I was considering creating the function.
But I would like to focus on this.
So, I'm going to create an issue related to the part you mentioned and implement the function later.